From 2dda7812c0bc29ec3d180df2dd3d575ad5d71b9d Mon Sep 17 00:00:00 2001
From: Arthur Freund <arthur.freund@online.de>
Date: Tue, 21 Feb 2017 16:04:33 +0100
Subject: [PATCH] Add compile flag options for quark

---
 configure | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure b/configure
index 6a50e5e..5b85347 100755
--- a/configure
+++ b/configure
@@ -912,6 +912,7 @@ def configure_v8(o):
   o['variables']['v8_optimized_debug'] = 0  # Compile with -O0 in debug builds.
   o['variables']['v8_random_seed'] = 0  # Use a random seed for hash tables.
   o['variables']['v8_use_snapshot'] = 'false' if options.without_snapshot else 'true'
+  o['variables']['v8_target_arch']= 'x87'
   o['variables']['node_use_v8_platform'] = b(not options.without_v8_platform)
   o['variables']['node_use_bundled_v8'] = b(not options.without_bundled_v8)
   o['variables']['force_dynamic_crt'] = 1 if options.shared else 0
-- 
2.7.4
